### ISSUE TITLE
A question about Fig 2 in the paper.

### ISSUE THREAD

**Issue by Zhengyang1995** [shjo-april/MARS#1]
Firstly, thank you for publishing such great work.

I have a question about the difference between Figure 2 (b) and (c) in the paper.

If I understand correctly, the results is separated by two feature cluster centroids made by K-means (as your setting, Kfg = 2).
For (c), result is only segmented in the region {(y, x)|Y^b_i(y, x) = c} which is come from the WSSS pseudo mask. 
Since in WSSS case we could use the same pseudo mask.why (b) is not segmented in the same region but the whole image?

and you mention Nc is the number of images for the mini-batch, and I didn't see the certain value for your setting in the Section 4.

Thank you for your time and impressive work.

**Comment by shjo-april** [comment]
Sorry for a late reply. 

1. We acknowledge your confusion related to Figure 2. We will replace the current figure with the intuitive figure.
2. Correctly, as Nc is the number of images for each class in all training dataset, we will update the notation.

I anticipate that we will release the final version including two revisions after preparing our camera-ready version.
Thank you for sharing your opinions!
